pkg/bundle/internal/target/target.go

```diff
@@ -396,3 +397,21 @@ func (b *Data) Populate(pool *util.CertPool, formats *trustapi.AdditionalFormats
 	}
 	return nil
 }
+
+func TruststoreHash(data []byte, bundle *trustapi.AdditionalFormats) string {
```

Comment:
```suggestion
func TrustBundleHash(data []byte, bundle *trustapi.AdditionalFormats) string {
```

I would suggest a more generic function name, as the hash is also used when no additional formats used. IMO a truststore is a binary format. The term trust bundle covers both PEM and additional formats (truststores).

Reply:
I updated the name.